script/blockchain/get_leaders_schedule.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require File.expand_path('../../config/environment', __dir__)
+
+NETWORKS.each do |network|
+  current_epoch = EpochWallClock.by_network(network).first&.epoch
+  if current_epoch
+    epochs_to_get = [current_epoch, current_epoch + 1]
+
+    epochs_to_get.each do |epoch|
+      puts "starting #{network} epoch #{epoch}"
```

Comment:
I'd move it to GetLeaderScheduleService. Otherwise this comment is logged every time, whether we pull data for this epoch or not.